sys/linux/key.txt

```diff
@@ -5,36 +5,40 @@ include <linux/key.h>
 include <linux/keyctl.h>
 include <uapi/linux/keyctl.h>
 
-resource key[int32]: 0, KEY_SPEC_THREAD_KEYRING, KEY_SPEC_PROCESS_KEYRING, KEY_SPEC_SESSION_KEYRING, KEY_SPEC_USER_KEYRING, KEY_SPEC_USER_SESSION_KEYRING, KEY_SPEC_GROUP_KEYRING, KEY_SPEC_REQKEY_AUTH_KEY, KEY_SPEC_REQUESTOR_KEYRING
+resource key[int32]: 0
+resource keyring[key]: KEY_SPEC_THREAD_KEYRING, KEY_SPEC_PROCESS_KEYRING, KEY_SPEC_SESSION_KEYRING, KEY_SPEC_USER_KEYRING, KEY_SPEC_USER_SESSION_KEYRING, KEY_SPEC_GROUP_KEYRING, KEY_SPEC_REQUESTOR_KEYRING
+resource user_key[key]
+resource reqkey_auth_key[key]: KEY_SPEC_REQKEY_AUTH_KEY
```

Comment:
I don't see reqkey_auth_key used anywhere. I think it will just not have any effect on fuzzing process if not used. I would just add KEY_SPEC_REQKEY_AUTH_KEY to keyring resource special values, it won't do harm if fuzzer pass it from time to time. I see that get_instantiation_keyring() explicitly filters out this value as invalid, if fuzzer passes it, it will be able to find cases where the values is not filtered but should.

Reply:
You can add a comment about why it is there to not confuse future readers.

Reply:
Okay, I put it under the keyring resource.